Source/ModuleSimpleBoiloff.cs

```diff
@@ -36,7 +36,7 @@ public class ModuleCryoTank: PartModule
         [KSPField(isPersistant = false)]
         public bool ShortwaveFluxAffectsBoiloff = false;
 
-        // Percent of insolation reflected
```

Comment:
Mmm, insolation is correct (incoming solar radiation)

Reply:
my bad, I assumed insulation